pkg/security/certManager/general/certManager.go

```diff
@@ -83,13 +83,13 @@ func New(config Config, logger *zap.Logger) (*CertManager, error) {
 		return nil, err
 	}
 	if err := c.watcher.Add(filepath.Dir(config.CAPool)); err != nil {
```

Comment:
Is this certwatcher working? Last experience is that cert-manager rotated certificates before expiration but our services didn't load the new file.

Reply:
Pls report as issue - i don't know, but as I remember that you report issue about change config file which we don't support.